packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -13,6 +13,15 @@
     "main": "dist/index.esm.js",
     "types": "dist/index.d.ts"
   },
+  "homepage": "https://backstage.io",
```

Comment:
I'm not sure this makes any sense in the template. Unless these can be customized via some template variables.